gitbook-docs/market/cmd-reference/user.md

```diff
@@ -1,22 +1,18 @@
 ---
```

Comment:
Headings:

# Creating new accounts
## Usage

python3 signup.py -h
python3 signup.py -u helloworld -p password123 -e hello@test.com

## Success

## Error: bla bla characters

## Error: duplicate username

Reply:
Edited this, is the current structure OK?

Reply:
Looks good.